hasura/migrations/default/1646251192439_create_table_public_vault_transactions/up.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE "public"."vault_transactions"
```

Comment:
maybe we can drop `date_type`, `posneg` and `active_users`, and make `value` a numeric type? and then add a column for details/description

Reply:
Ok, sounds good . what would we store in details/description?

Reply:
It could be for the Details column in the mockup